.github/workflows/test.yml

```diff
@@ -0,0 +1,53 @@
+name: Github Actions
+
+on: [push, pull_request]
+
+permissions:
+  contents: write
```

Comment:
I'm not sure this is correct. Why would we need write permission? Also doesn't this override any read permissions to the contents? Maybe that's also why the github actions aren't triggered in this PR?

Reply:
By the way, i think github actions won't run at this PR because in here what i'm doing is opening a PR against your current version of the code base, which does not include any github actions workflows. the triggers are only triggered when it is available on main code base. so for incoming PR it will execute as for my understanding

Reply:
regarding the permissions, let me run it without them to see if it executes on my forked repo.

Reply:
Okay i did remove the permissions and it worked without any errors on my repo, let me try by removing `shell=True` as well

![image](https://user-images.githubusercontent.com/61963664/193462968-1b159a5f-2414-46e8-a099-62d7e0c158f4.png)

Reply:
Okay so i ran without `shell=True` and it works fine on github actions, i think its something to be fixed on my local machine.